validator/src/bin/solana-test-validator.rs

```diff
@@ -42,6 +42,8 @@ use {
  */
 const DEFAULT_MAX_LEDGER_SHREDS: u64 = 10_000;
 
+const DEFAULT_FAUCET_SOL: f64 = 1_000_000.;
```

Comment:
bumping the default to 100m seems fine to me as well, 1m was completely arbitrary in the first place

Reply:
1M seems like a nice enough default. I'll let it ride for now